app/controllers/album.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const errors = require('../errors'),
+  { getAll } = require('../services/album'),
+  logger = require('../logger');
+
+exports.albumList = async (req, res, next) => {
+  try {
+    const albums = await getAll();
+    res.status(200).send(albums.data);
```

Comment:
`albums.data` should be done by service, because is a limitation of dependency used for build that service